war/src/main/less/modules/table.less

```diff
@@ -1,7 +1,7 @@
 .jenkins-table {
   --table-padding: 0.55rem;
 
-  width: calc(100% - 10px);
```

Comment:
![](https://i.imgur.com/gb2uJf0.png)
See right part of the table

Reply:
Awesome spot :) Should be fixed now if you wouldn't mind trying again :)